chia/util/keyring_wrapper.py

```diff
@@ -104,24 +106,37 @@ class KeyringWrapper:
     cached_passphrase_is_validated: bool = False
     legacy_keyring = None
 
-    def __init__(self, keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy: bool = False):
+    @classmethod
+    def create(
+        cls: Type[_T_KeyringWrapper], keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy: bool = False
+    ) -> Optional[_T_KeyringWrapper]:
         """
-        Initializes the keyring backend based on the OS. For Linux, we previously
-        used CryptFileKeyring. We now use our own FileKeyring backend and migrate
-        the data from the legacy CryptFileKeyring (on write).
+        Create a new KeyringWrapper instance.
         """
-        self.keys_root_path = keys_root_path
+        # instance: KeyringWrapper = KeyringWrapper()
+        instance = cls()
```

Comment:
is `cls` sometimes not `KeyringWrapper`?
If it always is, I think it would be clearer to just construct that type explicitly.

Reply:
Yeah, I am on the fence - naming the expected type seems better, because it is more constrained.
I'm not sure if the `cls()` helper does some extra Python class magic that I'm not aware of.

Reply:
@altendky do you have an opinion? In this case, I would tend to go with the more restrictive type, a Arvid is suggesting, but the usual thing in Python seems to be `cls()` - perhaps to avoid a class rename breaking code?